src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs

```diff
@@ -81,16 +85,25 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         {
             var releases = new List<ReleaseInfo>();
 
-            var categoryMapping = MapTorznabCapsToTrackers(query).Distinct();
-            var category = "0"; // Aka all
-            if (categoryMapping.Count() == 1)
+            var categoryMapping = MapTorznabCapsToTrackers(query).Distinct().ToList();
```

Comment:
Changing to `.FirstOrDefault()` removes multiple enumeration. We can drop `.ToList()` here for better performance.